src/apis/nimbus.json

```diff
@@ -49,6 +49,28 @@
         "description": "Get all the feature IDs from Nimbus.",
         "async": true,
         "parameters": []
+      },
+
+      {
+        "name": "getCurrentCollection",
+        "type": "function",
+        "description": "Get the current collection ID.",
+        "async": true,
+        "parameters": []
+      },
+
+      {
+        "name": "setCollection",
+        "type": "function",
+        "description": "Set the collection ID to the given collectionId.",
+        "async": true,
+        "parameters": [
+          {
+            "name": "collectionId",
+            "type": "string",
+            "description": "A string representing the collection to switch to."
```

Comment:
You don't need to mention it's a string. You can just say "The collection to switch to."